CRITICAL - 2022-06-16 10:26:19 --> Class "App\Models\UserModel" not found
#0 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#1 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#2 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-06-16 10:28:06 --> Class "App\Models\User_model" not found
#0 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#1 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#2 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-06-16 10:30:42 --> syntax error, unexpected token "class", expecting "function" or "const"
#0 C:\xampp\htdocs\myproject\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('auth')
#1 C:\xampp\htdocs\myproject\system\CodeIgniter.php(726): CodeIgniter\Router\Router->handle('auth')
#2 C:\xampp\htdocs\myproject\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-06-16 10:30:58 --> syntax error, unexpected token "class", expecting "function" or "const"
#0 C:\xampp\htdocs\myproject\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('auth')
#1 C:\xampp\htdocs\myproject\system\CodeIgniter.php(726): CodeIgniter\Router\Router->handle('auth')
#2 C:\xampp\htdocs\myproject\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-06-16 10:31:56 --> Class "App\Models\User_Model" not found
#0 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#1 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#2 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-06-16 10:32:34 --> Invalid file: login/index.php
#0 C:\xampp\htdocs\myproject\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login/index.php')
#1 C:\xampp\htdocs\myproject\system\Common.php(1089): CodeIgniter\View\View->render('login/index', Array, true)
#2 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(34): view('login/index')
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#4 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#5 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-06-16 10:35:15 --> Invalid file: login/index.php
#0 C:\xampp\htdocs\myproject\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login/index.php')
#1 C:\xampp\htdocs\myproject\system\Common.php(1089): CodeIgniter\View\View->render('login/index', Array, true)
#2 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(34): view('login/index')
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#4 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#5 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-06-16 10:35:17 --> Invalid file: login/index.php
#0 C:\xampp\htdocs\myproject\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login/index.php')
#1 C:\xampp\htdocs\myproject\system\Common.php(1089): CodeIgniter\View\View->render('login/index', Array, true)
#2 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(34): view('login/index')
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#4 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#5 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-06-16 10:39:08 --> Invalid file: login/index.php
#0 C:\xampp\htdocs\myproject\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login/index.php')
#1 C:\xampp\htdocs\myproject\system\Common.php(1089): CodeIgniter\View\View->render('login/index', Array, true)
#2 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(34): view('login/index')
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#4 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#5 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-06-16 10:39:11 --> Invalid file: login/index.php
#0 C:\xampp\htdocs\myproject\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login/index.php')
#1 C:\xampp\htdocs\myproject\system\Common.php(1089): CodeIgniter\View\View->render('login/index', Array, true)
#2 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(34): view('login/index')
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#4 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#5 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-06-16 10:39:43 --> Invalid file: login/index.php
#0 C:\xampp\htdocs\myproject\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login/index.php')
#1 C:\xampp\htdocs\myproject\system\Common.php(1089): CodeIgniter\View\View->render('login/index', Array, true)
#2 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(34): view('login/index')
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#4 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#5 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
ERROR - 2022-06-16 10:40:58 --> Table 'ci4.tb_login' doesn't exist
CRITICAL - 2022-06-16 10:40:58 --> Table 'ci4.tb_login' doesn't exist
#0 C:\xampp\htdocs\myproject\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\myproject\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\myproject\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\myproject\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\myproject\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\myproject\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(14): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#8 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#9 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-06-16 10:41:16 --> Undefined array key "user_id"
#0 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 21)
#1 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\auth->index()
#2 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\auth))
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2022-06-16 11:18:30 --> Column 'npm' cannot be null
CRITICAL - 2022-06-16 11:18:30 --> Column 'npm' cannot be null
#0 C:\xampp\htdocs\myproject\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `Da...', 0)
#1 C:\xampp\htdocs\myproject\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `Da...')
#2 C:\xampp\htdocs\myproject\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `Da...')
#3 C:\xampp\htdocs\myproject\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `Da...', Array, false)
#4 C:\xampp\htdocs\myproject\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\myproject\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\myproject\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\myproject\app\Controllers\Pages.php(33): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\Pages->save()
#9 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pages))
#10 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-06-16 11:56:46 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(15): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 15)
#1 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\Auth->index()
#2 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-06-16 11:58:06 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\myproject\app\Controllers\Auth.php(15): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 15)
#1 C:\xampp\htdocs\myproject\system\CodeIgniter.php(832): App\Controllers\Auth->index()
#2 C:\xampp\htdocs\myproject\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\myproject\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\myproject\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\myproject\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
